scipy-1.0/ckdtree.tex

```diff
@@ -45,21 +41,14 @@ \subsubsection{sparse distance matrix}
 sparse matrix to be directly converted to the common CSR, CSC, and COO
 data structures.
 
-
-\subsubsection{Pair-Counting}
-
-The cKDTree module implements the dual tree counting algorithm in (cite).
-
-An improvement to the pair counting algorithm is added to improve the scale
-with the number of bins.
+The cKDTree module implements a dual tree counting algorithm\cite{pmlr-v28-curtin13},
```

Comment:
let's see if one of @rainwoodman or @sturlamolden can confirm whether this is an appropriate reference or not

Reply:
@rainwoodman [Indicated](https://github.com/scipy/scipy-articles/pull/18#issuecomment-400831802) that:

> Gray and Moore are the dual tree algorithm indeed. 

So I should switch to that reference, which is [here](https://arxiv.org/abs/astro-ph/0012333)